MgidX: add optional host param

xmgiddev opened the pull request:
doc - https://github.com/prebid/prebid.github.io/pull/4932

gargcreation1992 commented on static/bidder-info/mgidX.yaml:
```diff
@@ -1,4 +1,4 @@
-endpoint: "https://us-east-x.mgid.com/pserver"
+endpoint: "https://{{.Host}}.mgid.com/pserver"
```
This change is not recommended. We highly discourage folks to use dynamic domain this way. Please review - https://github.com/prebid/prebid-server/issues/2606

onkarvhanumante replied:
@xmgiddev requesting to address above comment

gargcreation1992 reviewed (changes requested):
Please review https://github.com/prebid/prebid-server/issues/2606

onkarvhanumante commented:
@xmgiddev, prebid server had major release 2.0

requesting to update `mgid:master` with latest prebid master

onkarvhanumante commented:
> @xmgiddev, prebid server had major release 2.0
> 
> requesting to update `mgid:master` with latest prebid master

@xmgiddev  requesting to update `mgid:master` with latest prebid master and address PR comment

onkarvhanumante commented:
> > @xmgiddev, prebid server had major release 2.0
> > requesting to update `mgid:master` with latest prebid master
> 
> @xmgiddev requesting to update `mgid:master` with latest prebid master and address PR comment

@xmgiddev ping for updating this PR

bretg reviewed (changes requested):
Sorry, but requiring users to enter a region is not acceptable.

I'll write up the solution in a more formal way but here's a summary:
- document for host companies that you request regional endpoints
- when they deploy your adapter's YAML file, they are responsible for using an appropriate regional endpoint

It is unrealistic to expect that multinational publishers can sort of which endpoint a given host should use. This places a heavy (or impossible) burden on the implementation of websites.

onkarvhanumante commented:
closing this inactive PR. can we reopened when comments gets addressed

xmgiddev commented:
Sorry for not answering for a long time
@bretg we have changed the required HOST param to the optional REGION param
@onkarvhanumante we updated with latest prebid master

bretg commented:
@xmgiddev - please open a new PR from your fork.
